requirements.txt

```diff
@@ -2,8 +2,8 @@ matplotlib==3.0.3
 google_images_download==2.5.1
 numpy==1.16.2
 keras==2.2.4
-scikit_image==0.15.0
-scikit_learn==0.20.3
+scikit-image==0.15.0
```

Comment:
Are you getting any errors?

It appears that both

```
pip install scikit-image
```

and 

```
pip install scikit_image
```

are valid installation options